chimedb/data_index/orm.py

```diff
@@ -421,9 +446,31 @@ class WeatherFileInfo(base_model):
     DigitalGainFileInfo,
     CalibrationGainFileInfo,
     FlagInputFileInfo,
+    MiscFileInfo,
 ]
 
 
+class MiscFileList(base_model):
```

Comment:
I wonder if we really want to bother indexing the files within a misc file. Things like the RFI ones will have tens of thousands of files, and they'll all have the same size and names that aren't that interesting. For those anyway, I can't really see us wanting to look through them all. I think people should be encouraged to keep enough information at the misc file level to know what's inside the tar file.

Reply:
Ah, yeah, I wasn't think about RFI dumps.  If there's going to be tens of thousands of files per tarfile, probably best to just do without.